### PR TITLE
Add EIP: System logs

### PR DESCRIPTION
**Pull request by etan-status**

### CODE REVIEW

File: EIPS/eip-####.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
+---
```
Comment (abcoathup) [inline EIPS/eip-####.md:1]:
```suggestion
---
eip: 7799
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.

File: EIPS/eip-####.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+title: System logs
+description: Per-block logs without associated transactions
+author: Etan Kissling (@etan-status), Gajinder Singh (@g11tech)
+discussions-to: https://ethereum-magicians.org/t/eip-system-logs/21497
```
Comment (abcoathup) [inline EIPS/eip-####.md:5]:
```suggestion
discussions-to: https://ethereum-magicians.org/t/eip-7799-system-logs/21497
```
I added EIP number to the Eth Magicians title

**Review by g11tech — APPROVED** [review]
lgtm

**Review by Magken91 — APPROVED** [review]

**Review by Cplus360 — COMMENTED** [review]
Nice work reviewing

**Review by Cplus360 — COMMENTED** [review]
Viewing

**Pull request merged**